server/service/osquery_utils/queries.go

```diff
@@ -1008,16 +1012,17 @@ func directIngestOSWindows(ctx context.Context, logger log.Logger, host *fleet.H
 		Arch:          rows[0]["arch"],
 		KernelVersion: rows[0]["kernel_version"],
 		Platform:      rows[0]["platform"],
+		Version:       rows[0]["version"],
 	}
 
-	version := rows[0]["version"]
-	if version == "" {
+	displayVersion := rows[0]["display_version"]
+	if displayVersion == "" {
 		level.Debug(logger).Log(
 			"msg", "unable to identify windows version",
 			"host", host.Hostname,
 		)
 	}
-	hostOS.Version = version
+	hostOS.DisplayVersion = displayVersion
```

Comment:
looking back in the git history, it looks like the original intent here was to debug the "display_version" (eg. "22H2"), not the current implementation that uses the "build version" (eg. 10.0.22000) which is why I'm replacing here.  I suspect this may have been buggy in some older operating systems.